shared/usingSupport/_default.md

```diff
@@ -1,5 +1,8 @@
-If at anytime while using resin.io you find yourself stuck, confused or alone remember that there is a team of helpful and friendly support engineers just one click away. There are 3 places to chat to us:
-* Click the big yellow "?", bottom right of your dashboard for realtime chat with our support engineers.
-* Post your questions in the [Troubleshooting section](https://forums.resin.io/c/troubleshooting) of the [resin.io forums](https://forums.resin.io/).
-* Head over to our community chat at [gitter.im](https://gitter.im/resin-io/chat), maybe also make some friends!
-* Good ol' fashion email, at support@resin.io
+If at anytime while using resin.io you find yourself stuck, confused or alone remember that there is a team of helpful and friendly support engineers just one click away. 
+
+The best place to chat with us is in the [resin.io forums](https://forums.resin.io). Post your questions to the [Troubleshooting section](https://forums.resin.io/c/troubleshooting), where our engineers can help address any issues you may be having with resin.io.  
+
+We also love to meet users in our community chat at [gitter.im](https://gitter.im/resin-io/chat).
+
+And there's always good ol' fashioned email, at hello@resin.io.
```

Comment:
I think this would push free users to asking support questions (stuck, confused, alone questions) to hello@. If we want to show this email address at this point of the doc, we could change the language to indicate what kind of emails we want there. I can't think of why we'd want users, at this point in the doc, to email hello@.

We could add a message about "If you'd like more direct support from our engineers, check out our support pricing plan."